src/main/java/org/myrobotlab/service/Proxy.java

```diff
@@ -7,6 +7,7 @@
 import org.myrobotlab.framework.Registration;
 import org.myrobotlab.framework.Service;
 import org.myrobotlab.framework.ServiceType;
+import org.myrobotlab.framework.repo.ServiceData;

```

Comment:
Why do we have service Data when all this info is supposed to be on the MetaData object?

Reply:
Tried to merge, got slapped for too many changes.